examples/dotnet/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.3"
+  required_version = ">= 1.5.0"
```

Comment:
Would this be a breaking change for customers?

Reply:
In theory, yes for anyone using Terraform versions 1.3.x or 1.4.x. But the breaking change was actually in a previous commit where I added the `check` block, which is only supported on versions 1.5.0 and up.

https://github.com/DataDog/terraform-aws-lambda-datadog/pull/6

What's the best way to handle these types of breaking changes moving forward?

Reply:
Chatted offline, updated v1.1 release to reflect this breaking change and will do the same when this is released

https://github.com/DataDog/terraform-aws-lambda-datadog/releases/tag/v1.1.0